libs/shared/src/consts/password-helper/passwordHelper.ts

```diff
@@ -0,0 +1,5 @@
+export const passwordConstraints = {
+  minLength: 8,
+  maxLength: 64,
+  pattern: /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]{8,64}$/,
```

Comment:
Seeing the RegExp I think we should inform the users which are the valid special characters. Some of them are missing and users can receive an error message and be confused about it.

Reply:
You're right, will add them to the error message

Reply:
There is this example in [Mantine to show password strength ](https://mantine.dev/core/password-input/#strength-meter-example)
I kind of like that it shows requirements separately. Let me know if its any help @BiswaViraj  :)
<img width="527" alt="Screen Shot 2022-12-18 at 13 37 09" src="https://user-images.githubusercontent.com/7778680/208296209-27a9a895-b1e7-4cf2-882c-d06204bef35a.png">

Reply:
Yeah I think we can use a similar design will just replace the bar with the strength meter
Thanks @ainouzgali